Description: Upstream generates a licensing report for Jenkins deps.
 Not generated during debian build so excluding from About Jenkins page.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: jenkins/core/src/main/resources/hudson/AboutJenkins/index.jelly
===================================================================
--- jenkins.orig/core/src/main/resources/hudson/AboutJenkins/index.jelly	2012-02-10 15:39:22.749867338 +0000
+++ jenkins/core/src/main/resources/hudson/AboutJenkins/index.jelly	2012-02-28 16:48:45.517724331 +0000
@@ -31,13 +31,6 @@
       <p>
         ${%blurb}
       </p>
-
-      <p>
-        ${%dependencies}
-      </p>
-      <t:thirdPartyLicenses>
-        <st:include page="/META-INF/licenses.xml" optional="true" />
-      </t:thirdPartyLicenses>
     </l:main-panel>
   </l:layout>
 </j:jelly>
Index: jenkins/core/src/main/resources/hudson/AboutJenkins/index.properties
===================================================================
--- jenkins.orig/core/src/main/resources/hudson/AboutJenkins/index.properties	2012-02-10 15:39:22.749867338 +0000
+++ jenkins/core/src/main/resources/hudson/AboutJenkins/index.properties	2012-02-28 16:48:45.517724331 +0000
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-about=About Jenkins {0}
+about=About Jenkins {0} (Debian/Ubuntu distribution)
 blurb=<a href="http://jenkins-ci.org/">Jenkins</a> is a community-developed open-source continuous integration server.
 
-dependencies=Jenkins depends on the following 3rd party libraries.
\ No newline at end of file
+dependencies=Jenkins depends on the following 3rd party libraries.
